.github/scripts/check_facility_has_institution_id/main.py

```diff
@@ -0,0 +1,54 @@
+# Run through all the FACILITY.yaml files and check that they have an institution_id field.
+
+import yaml
+import sys
+import glob
+import requests
+
+export_mapping = False
+
+# Get the list of valid institution ids
+response = requests.get("https://topology-institutions.osg-htc.org/api/institution_ids")
+topology_institutions = response.json()
+topology_institutions_by_id = {x['id']: x for x in topology_institutions}
+topology_institution_ids = {x['id'] for x in topology_institutions}
+
+def check_facility_institution_id(yaml_string: str):
+
+    facility = yaml.load(yaml_string, Loader=yaml.Loader)
+    if 'InstitutionID' not in facility:
+        raise Exception("FACILITY.yaml does not have an InstitutionID field")
+
+    if facility['InstitutionID'] not in topology_institution_ids and facility['InstitutionID'] is not None:
+        raise Exception(f"Invalid InstitutionID: {facility['InstitutionID']}")
```

Comment:
Looks like this script bails out on the first error. Can you make it print all errors instead and return nonzero if there were any?